src/js/components/Articles/ArticleButtons.scss

```diff
@@ -0,0 +1,94 @@
+.reader-buttons {
+  margin: 0 auto;
+  margin-top: 544px;
+  max-width: 400px;
+  border-radius: 50%;
+}
+.videoplayer-buttons {
+  margin: 0 auto;
+  margin-top: 50px;
+  max-width: 400px;
+  border-radius: 50%;
+}
+
+//@import "compass/css3";
+
+// Variables
+$gray: #808080;
+$yellow: #c3ec0b;
+
+// Mixin
+@mixin circle-btn($bkgd-color, $font-size) {
+  color: #333937;
+  font-family: "lato";
+  font-weight: bold;
+  display: inline-block;
+  text-decoration: none;
+  font-size: $font-size;
+  margin: 0.5em;
+  width: 5em;
+  height: 5em;
+  border-radius: 50%;
+  padding: 1.45em 0;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+  -webkit-transition: all 0.2s ease-in-out;
+  -moz-transition: all 0.2s ease-in-out;
+  -o-transition: all 0.2s ease-in-out;
+  -ms-transition: all 0.2s ease-in-out;
+  transition: all 0.2s ease-in-out;
+  background: $bkgd-color;
+  &:after {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 4em;
+    height: 4em;
+    border-radius: 30%;
+    background: rgba(255, 255, 255, 0.1);
+    box-shadow: 0 0 3em 0 rgba(255, 255, 255, 0.1);
+    top: 130%;
+    left: 0;
+    z-index: 1;
+    -webkit-transition: all 0.2s ease-in-out;
+    -moz-transition: all 0.2s ease-in-out;
+    -o-transition: all 0.2s ease-in-out;
+    -ms-transition: all 0.2s ease-in-out;
+    transition: all 0.2s ease-in-out;
+  }
+  &:hover {
+    border-radius: 30%;
+    &:after {
+      top: 0;
+    }
+  }
+  &:active {
+    &:after {
+      background: rgba(0, 0, 0, 0.1);
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+    }
+  }
+}
+
+// Styling
+* {
+  box-sizing: border-box;
+}
+body {
+  padding: 0px;
```

Comment:
Why 0 padding ?

Reply:
The main reason this is used is because sometimes the browser applies its default padding on the body.
The default padding should be 8 px if I'm not mistaken.
Making it 0 px ensures that you're pretty sure that the body will is initialized to 0.

Reply:
sorted by removing it completely